webpack.conf.js

```diff
@@ -1,28 +1,71 @@
-var webpack = require('webpack');
+var CleanWebpackPlugin = require('clean-webpack-plugin'),
+    config = require('./config.json'),
+    HtmlWebpackPlugin = require('html-webpack-plugin'),
+    path = require('path'),
+    Visualizer = require('webpack-visualizer-plugin'),
+    webpack = require('webpack');
 
 module.exports = {
+    watch: false,
     devtool: 'source-map',
+    context: __dirname,
+    entry: {
+        main: './assets/js/app.js'
+    },
+    output: {
+        filename: '[chunkhash].theme-bundle.[name].js',
+        path: path.resolve(__dirname, "assets/dist"),
+        publicPath: 'assets/dist/', // TODO: cdnify - https://github.com/bigcommerce/paper/blob/master/index.js#L154
```

Comment:
@mcampa - i'm wondering how we can make this work here.  we want the publicPath to include the CDN for the code-split files.  here's some more info:  https://webpack.js.org/guides/public-path/